files/private-chef-cookbooks/private-chef/templates/default/sv-keepalived-run.erb

```diff
@@ -1,3 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
hey @stevendanna does this change feel gratuitous to you?   tickling keepalived always has me nervous